app/src/routes/examples/example7/+page.svelte

```diff
@@ -1,22 +1,35 @@
 <script lang="ts">
-import data from '../../../data/jobs.json';
+import originalData from '../../../data/jobs.json';
 import PowerTable from '$lib/components/PowerTable.svelte';
 import type { Data, Options, Instructs } from '$lib/components/PowerTable.svelte';
+import MyComponent from './MyComponent.svelte';
+import type { ComponentType, SvelteComponent } from 'svelte';
 
 let myPowerTable: PowerTable;
 
 let ptInstructs: Instructs[] = [];
 
-// Here we set the instructs automatically based on the first row of data and set parseAs to 'html' for all instruct (we could do this manually too. with parseAs set to 'html', we can edit the cells and see how it handles HTML tags)
-let tempInstructs: Instructs[] = [];
-Object.keys(data[0]).forEach(key => {
-    tempInstructs.push({
-        key: key,
-        title: key,
-        parseAs: 'html',
-    });
-});
-ptInstructs = tempInstructs;
+let data = JSON.parse(JSON.stringify(originalData))
+// Here we set the instructs to make `company` field parsed as HTML, and `department` to be edited as a dropdown instead of a textarea
+let uniqueDepartments = new Set();
+data.forEach(row => uniqueDepartments.add(row['department']))
+ptInstructs = [
+    {key: 'id'},
+    {key: 'first_name'},
+    {key: 'last_name'}, 
+    {key: 'company', parseAs: 'html'}, // with parseAs set to 'html', we can edit the cells and see how it handles HTML tags
+    {
+        key: 'department', 
+        edit: {
+            as: <ComponentType<SvelteComponent>>MyComponent,
+            props: {selectValues: uniqueDepartments}
+        },
+    },
+    {key: 'job'}
+];
+
+// default a company field to HTML to make capabilities more clear
+data[0]['company'] = `<li>${data[0]['company']}</li><li>html supported</li>`
```

Comment:
This might not be the right example page for this demonstration. But more importantly, in many cases it's better not to mix HTML tags with the underlying data if all we want is a better styling. HTML can interfere with text processing (e.g. search and filter). If you change a cell value from `John` to `<li>John</li>` and then filter/search for `li` that cell will show up which is usually not expected.

A proper way to do this for your example is to add a break line between `${data[0]['company']}` and `html supported` and then use a `customParse` to wrap each line in `<li>` tags. This way the HTML tag would not mix with the data.

Reply:
> This might not be the right example page for this demonstration. 

Agreed. I was trying to keep the same functionality of the example before (with the addition of making it clear when viewing the page that the Company column allowed HTML). But seems appropritate to remove since `parseAs: 'html` is used in other examples.

> A proper way to do this for your example...

Awesome, I appreciate you taking the time to teach me a bit here and provide reasoning!